### ISSUE TITLE
Shift+Enter Newline

### ISSUE THREAD

**Issue by markus-gx** [storyblok/monoblok#23]
[storyblokurl]: https://www.storyblok.com?utm_source=github.com&utm_medium=issue-template

When migrating from the deprecated richTextResolver to the new one `storyblok/richtext`, text's with a `SHIFT+Enter` (newline) do have a separate new line now. 

## Expected Behavior

Just one `<br/>` tag instead of two. 

## Current Behavior

Two `br`-tags are generated. I guess the one with a `key` attribute are correct, since the new RichTextResovler works like that, but the other one is incorrect. 
```
<br key="br-4">
<br>
text
<br key="br-6">
<br>
```

**Comment by dipankarmaikap** [comment]
@markus-gx I don't think this issue is present anymore, so I'm closing it for now. Feel free to reopen it if you think it still exists.
